SU-SWS/decanter
# Adapt-1808: Use TW box shadows for v7

# READY FOR REVIEW

# Summary
- Get rid of our custom box shadows and use TW default
- Update our button styles to use TW md drop shadow because that's closer to our previous default
- Update preview page

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. See the shadows: https://deploy-preview-799--decanter-v7.netlify.app/#drop-shadows
1. Look at code

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
